glusterd2/peer/store-utils.go

```diff
@@ -210,3 +210,12 @@ func GetPeerIDByAddr(addr string) (uuid.UUID, error) {
 	}
 	return p.ID, nil
 }
+
```

Comment:
See earlier comment, this should be a method of the type and named `MetadataSize()`